tests/persistence_tests/test_shamir_persistence.py

```diff
@@ -44,7 +45,10 @@ def test_abort(core_emulator: Emulator):
 
     assert features.recovery_status == RecoveryStatus.Nothing
 
-    device_handler.run(device.recover, pin_protection=False)
+    session = device_handler.client.get_seedless_session()
+    device_handler.run_with_provided_session(
+        session, device.recover, pin_protection=False
+    )
 
     recovery.confirm_recovery(debug)
```

Comment:
There seems to be an existing race between running `device.recover` in a thread and confirming that the recovery layout has been launched, causing the following test failures:
https://github.com/trezor/trezor-firmware/actions/runs/16174356159/job/45656150594#step:6:235